repo-issues/lib/data-source.js

```diff
@@ -19,36 +19,51 @@ const generateViewState = (configuration) => {
     showPRs = configuration.showPRs
 
     octokit.authenticate({type: 'oauth', token: `${configuration.token}`})
-    
-    return octokit.repos.getForOrg({ org: organisation, type: 'all', per_page: 9999 })
-        .then(repos => Promise.all(repos.data.map(queryRepoIssues)))
-        .then(prs => prs.filter(each => each.prs.length > 0 || each.issues.length > 0))
+
+    return paginate(octokit.repos.getForOrg)
+        .then(data => Promise.all(data.map(queryRepoIssues)))
+        .then(repos => repos.filter(each => each.prs.length > 0 || each.issues.length > 0)) // remove repositories with no open prs or issues
+        .then(repos => repos.sort((a, b) => a.pushed_at < b.pushed_at)) // sort repositories by last pushed date
         .then(toViewState)
 }
 
+function paginate(method) {
+    return method({ org: organisation, type: 'public', per_page: 100 })
+        .then(response => {
+            let data = response.data
+            while (octokit.hasNextPage(response)) {
+                return octokit.getNextPage(response).then(response => data.concat(response.data))
+            }
+            return data
+        })
+}
+
+
```

Comment:
3 empty lines?

Reply:
Sorry I have a script to automatically insert random empty spaces at the end of the file to drive @takecare crazy  😛  
Fixing!